lib/src/main/java/com/diffplug/spotless/java/RemoveWildcardImportsStep.java

```diff
@@ -20,13 +20,17 @@
 
 /** Removes any wildcard import statements. */
 public final class RemoveWildcardImportsStep {
+
+	/**
+	 * Matches lines like 'import foo.*;' or 'import static foo.*;'.
+	 */
+	private static final String REGEX = "(?m)^import\\s+(?:static\\s+)?[^;\\n]*\\*;\\R?";
+	private static final String NAME = "removeWildcardImports";
+	private static final String ERROR = "Do not use wildcard imports. 'spotlessApply' cannot resolve this issue.";
```

Comment:
How about
```suggestion
	private static final String ERROR = "Do not use wildcard imports (e.g. java.util.*) - replace with specific class imports (e.g. java.util.List) as 'spotlessApply' cannot auto-fix this";
```
or we can remove the examples from the error message